app/assets/stylesheets/admin/variables.css.scss

```diff
@@ -8,3 +8,5 @@ $warning-red: #da5354;
 $warning-orange: #da7f52;
 $medium-grey: #919191;
 $pale-blue: #cee1f4;
+
+$admin-enterprises-index__admin-actions-divider-color: #cee1f4;
```

Comment:
...and this can be deleted.

Reply:
@Matt-Yorkley Agh. Thanks, I was switching between colors and didn't realize that the final one I picked was declared as a variable immediately above.

What do you think about still using the variable for the divider color, but setting it `$pale-blue`, like this:

```css
$admin-enterprises-index__admin-actions-divider-color: $pale-blue;
```

In my experience, separating colors would make it easier to make color changes site-wide. Especially for OFN which is multi-instance, I imagine it would be convenient if more colors were assigned like this.

Reply:
I don't think the variable for `$admin-enterprises-index__admin-actions-divider-color` needs to be shared throughout all the SCSS, as it's a single element on a single page.

I'm not sure about the complex name either, it seems much less likely to be re-used elsewhere if it's named by the exact page it's in.

Reply:
I know what you mean though. I think as the colour is the same as the border colour for tables, I think I'd probably add an entry like this with the main colours:
```
$admin-table-border: $spree-blue;
```
then use that on your divider element:
```
background-color: $admin-table-border;
```